storage/Cargo.toml

```diff
@@ -35,6 +35,7 @@ url = { version = "2.1.1", optional = true }
 vm-memory = "0.10"
 fuse-backend-rs = "^0.10.3"
 gpt = { version = "3.0.0", optional = true }
+toml = "0.5"
```

Comment:
Maybe move it to `[dev-dependencies]`.

Reply:
Thanks for your review. Now, I have fixed it with new commit.